### DISCUSSION TITLE
How to send a search query from the Search Vertical to Search Results?

### DISCUSSION THREAD

**Discussion by Fringie** [microsoft-search/pnp-modern-search#2594] [Q&A] [answered]
Hi,

I want to use Search Verticals to filter data in the Search Results page. Can the query be stored in the Search Vertical, which is then sent to the search results?

For example:

I have Vertical tabs called:
- Everything
- HR
- Accounting

If the user selects the "Everything" Vertical, then the query * is run to return all the search results
If the user selects the "HR" Vertical, then the query searches for * and HR to return HR-specific data.
If the user selects the "Accounting" Vertical, then the query searches for * and Accounting to return Accounting-specific data.

The way the webparts seem to work, is I would need to add three search results webparts, then the Verticals display the relevant search results webpart which seems clunky i.e. I'll likely have 20 search verticals which means I'd need 20 search results webpart on my page

**Comment by wobba** [comment, marked as answer]
Did you look at the docs? https://microsoft-search.github.io/pnp-modern-search/usage/search-results/connections/#verticals And correct, one result web part per vertical.

And also tutorials if you have more pages, and not multiple parts on a page. https://microsoft-search.github.io/pnp-modern-search/scenarios/Create-a-search-page-with-verticals-on-different-pages/

**Reply by Fringie** [reply]
Hi,

I thought I had read the documentation, but I must have missed this page. This answered my question - I needed to use {Vertical.value}
